docs/UserGuide.md

```diff
@@ -117,23 +117,75 @@ Examples:
 * `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 * `edit 3 sch/NJC stream/` Edits the school of the 3rd person to be `NJC` and clears academic stream data.
 
-### Locating persons by name: `find`
+### Finding students by fields: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds all students whose fields match the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find [n/KEYWORD …] [a/KEYWORD …] [e/KEYWORD …] [p/KEYWORD …] [sch/SCHOOL] [stream/ACAD_STREAM] [lvl/ACAD_LEVEL]`
```

Comment:
instead of just `KEYWORD`, maybe it will be clearer if we put `[n/NAME_KEYWORD] [a/ADDRESS_KEYWORD]` etc?

Reply:
it's a bit long though

Reply:
how about i just ditch the KEYWORD part